### PR TITLE
Release notes for 2.19.8

### PR DESCRIPTION
**Pull request by bunchesofdonald**
Release notes for 2.19.8

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+- Handle join edge case in `ParameterTypeError.from_validation_error` — https://github.com/PrefectHQ/prefect/pull/14420
+- Respect `--ui/--no-ui` flag on `prefect server start` — https://github.com/PrefectHQ/prefect/pull/14520
+- Append task run futures only when entering task run engine from flow run context — https://github.com/PrefectHQ/prefect/pull/14439
+
+## New Contributors
+* @oaustegard made their first contribution in https://github.com/PrefectHQ/prefect/pull/14505
+
+### Integrations
+- Depend on Cloud Run Execution state for timeout enforcement 2.x — https://github.com/PrefectHQ/prefect/pull/14554
+- Handle case where `AwsClientParameters` is a dict — https://github.com/PrefectHQ/prefect/pull/14438
```
Comment (billpalombi) [inline RELEASE-NOTES.md:18]:
```suggestion
- Handle case where `AwsClientParameters` is a `dict` — https://github.com/PrefectHQ/prefect/pull/14438
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+- Added print_func for run_namespaced_job — https://github.com/PrefectHQ/prefect/pull/14463
```
Comment (billpalombi) [inline RELEASE-NOTES.md:19]:
```suggestion
- Added `print_func` for `run_namespaced_job` — https://github.com/PrefectHQ/prefect/pull/14463
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+- Use correct paths in `GcsBucket.put/get_directory` — https://github.com/PrefectHQ/prefect/pull/14537
+- Cache kubernetes client — https://github.com/PrefectHQ/prefect/pull/14535
```
Comment (billpalombi) [inline RELEASE-NOTES.md:21]:
```suggestion
- Cache Kubernetes client — https://github.com/PrefectHQ/prefect/pull/14535
```

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,28 @@
...
+- Remove old WorkQueue components. — https://github.com/PrefectHQ/prefect-ui-library/pull/2601
```
Comment (billpalombi) [inline RELEASE-NOTES.md:22]:
```suggestion
- Remove old WorkQueue components — https://github.com/PrefectHQ/prefect-ui-library/pull/2601
```

Comment (desertaxle) [inline RELEASE-NOTES.md:22]: This one looks like it should be under a UI heading

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,28 @@
...
+### Enhancements
+- Allow loading flows from an entrypoint when dependencies are missing — https://github.com/PrefectHQ/prefect/pull/14548
+
+### Fixes
+- Handle join edge case in `ParameterTypeError.from_validation_error` — https://github.com/PrefectHQ/prefect/pull/14420
+- Respect `--ui/--no-ui` flag on `prefect server start` — https://github.com/PrefectHQ/prefect/pull/14520
+- Append task run futures only when entering task run engine from flow run context — https://github.com/PrefectHQ/prefect/pull/14439
+
+## New Contributors
+* @oaustegard made their first contribution in https://github.com/PrefectHQ/prefect/pull/14505
```
Comment (desertaxle) [inline RELEASE-NOTES.md:14]: Think generally the new contributors go below all the PRs

**Review by desertaxle — APPROVED** [review]

File: RELEASE-NOTES.md (hunk 5 above)
Comment (bunchesofdonald) [reply RELEASE-NOTES.md:14]: 👍 This is how the script output them, I'll update it too.

**Review by billpalombi — APPROVED** [review]

File: RELEASE-NOTES.md (hunk 5 above)
Comment (bunchesofdonald) [reply RELEASE-NOTES.md]: Looks difficult to change the script, I'll make an issue for it.

**Pull request merged**